packages/@cdktf/provider-generator/lib/get/generator/emitter/resource-emitter.ts

```diff
@@ -52,6 +55,29 @@ export class ResourceEmitter {
     );
   }
 
+  private emitStaticMethods(resource: ResourceModel) {
+    const comment = sanitizedComment(this.code);
+    comment.line(
+      `For generation of configuration for import, run "cdktf plan <stack-name>"`
```

Comment:
I was thinking of something more like, but I'm also no good docs editor 😅

```suggestion
      `Creates an import block that will generate CDKTF code for importing a ${resource.className} resource upon running "cdktf plan <stack-name>"`
```

Reply:
Included 👍